scripts/SE.js

```diff
@@ -428,10 +429,55 @@ SE = {
 		}
 
 		SE.LoadBalances(account, r => {
-			SE.ShowHomeView('balances', { balances: r, account: account }, { a: account });
+			SE.GetScotUserTokens(account, scotTokens => {
+				SE.ShowHomeView('balances', { balances: r, scotTokens: scotTokens, account: account }, { a: account });
+			});
 		});
 	},
 
+	GetScotUserTokens: function(account, callback) {
+		if (!account && SE.User) {
+			account = SE.User.name;
+		}
+
+		$.get(Config.SCOT_API + `@${account}`, { v: new Date().getTime() }, results => {
+			SE.User.ScotTokens = results;
+
+			if (callback) {
+				callback(Object.entries(results));
+			}
+		});
+	},
+
+	ClaimToken: function(symbol) {
+		SE.ShowLoading();
+		
+		const token = SE.Tokens.find(t => t.symbol === symbol);
+		const username = SE.User.name;
+		const amount = SE.User.ScotTokens[symbol].pending_token;
+		const factor = Math.pow(10, token.precision);
+		const calculated = amount / factor;
+
+		const claimData = {
+			symbol
```

Comment:
claimData should be
{"symbol": symbol}

Reply:
The way I've done it works, this is an ES6 feature which restructures the value, it makes it so you don't have to redundantly define nothing the key and property if they're the same name. I do this for the Scotbot initialisation and settings app as well.